src/signature_pad.ts

```diff
@@ -386,7 +390,11 @@ export default class SignaturePad extends SignatureEventTarget {
   };
 
   private _handlePointerMove = (event: PointerEvent): void => {
-    if (!this._isLeftButtonPressed(event, true) || !this._drawingStroke) {
+    if (
+      !event.isPrimary ||
```

Comment:
I think we should just return if !event.isPrimary adding it to this if will end the stroke. We should also return from _handlePointerUp if !event.isPrimary.